backend/src/main/java/com/capstone/moneytree/model/node/User.java

```diff
@@ -85,10 +101,16 @@ public void unfollow(User user) {
     }
 
     public void made(Transaction transaction) {
+        if (this.transactions == null) {
+            this.transactions = new ArrayList<>();
```

Comment:
We may have to use the strategy @EspressoCode made in the PDF document about a bug we had earlier with follow/unfollow. @EspressoCode what do you think?

Reply:
I agree, I wanted to change this logic in my frontend story for buy/sell, if @Razine don't mind I will rebase with these new changes and then change to that implementation explained in PDF.

Reply:
I dont mind at all bro, please, make sure to document the implementation because it is considered refactoring. We will need to include that change in the software architecture document in release 3 because our data model has changed.